RULES.md

```diff
@@ -136,7 +136,11 @@ There are currently no warnings for this rule pack.
 
 ### Errors
 
-There are currently no errors for this rule pack.
+| AWS Config Rule Name | Summary | Relevant Control IDs |
+| --- | --- | --- |
+| [cloud-trail-log-file-validation-enabled](https://docs.aws.amazon.com/config/latest/developerguide/cloud-trail-log-file-validation-enabled.html) | Utilize AWS CloudTrail log file validation to check the integrity of CloudTrail logs. Log file validation helps determine if a log file was modified or deleted or unchanged after CloudTrail delivered it. This feature is built using industry standard algorithms: SHA-256 for hashing and SHA-256 with RSA for digital signing. This makes it computationally infeasible to modify, delete or forge CloudTrail log files without detection.| 164.312(c)(1), 164.312(c)(2) |
+| [cloud-trail-encryption-enabled](https://docs.aws.amazon.com/config/latest/developerguide/cloud-trail-encryption-enabled.html) | Because sensitive data may exist and to help protect data at rest, ensure encryption is enabled for your AWS CloudTrail trails. | 164.312(a)(2)(iv), 164.312(e)(2)(ii) |
+| [cloud-trail-cloud-watch-logs-enabled](https://docs.aws.amazon.com/config/latest/developerguide/cloud-trail-cloud-watch-logs-enabled.html) | Use Amazon CloudWatch to centrally collect and manage log event activity. Inclusion of AWS CloudTrail data provides details of API call activity within your AWS account. | 164.308(a)(3)(ii)(A), 164.312(b) |
```

Comment:
Error/Warning names should match the rule name and the list should be alphabetical order

Ex.  `cloud-trail-log-file-validation-enabled` -> `HIPAA-Security-CloudTrailLogFileValidationEnabled`

Reply:
I also think we should follow the rule name format that we used in the NIST 800-53 Pack

`HIPAA-Security-CloudTrailLogFileValidationEnabled` -> `HIPAA.Security-CloudTrailLogFileValidationEnabled`

Thoughts?

Reply:
All sounds good. updating now